Nwazet.Commerce.Tests/Stubs/AuthorizationServiceStub.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Orchard.Security;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Orchard.ContentManagement;
+using Orchard.Security.Permissions;
+
+namespace Nwazet.Commerce.Tests.Stubs {
+    /// <summary>
+    /// this implemerntation used in tests for Territories
+    /// </summary>
+    public class AuthorizationServiceStub : IAuthorizationService {
+        public void CheckAccess(Permission permission, IUser user, IContent content) {
+            throw new NotImplementedException();
+        }
+
+        public bool TryCheckAccess(Permission permission, IUser user, IContent content) {
+            if (user == null || user.UserName == "admin") {
```

Comment:
Using a mock now.
It simulates the condition where an anonymous user has all permissions, just so that I could test something where there is something with all permissions.